Pipfile.lock

```diff
@@ -1,20 +1,20 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "3e250b03724612ccadb22ee7e20c5cf9563f8a0752f9374ec0d55b56fa55bebb"
+            "sha256": "572443aad8e855844d94fcecaa649f6f3b99137e429727b121115c695a32f0d6"
         },
         "host-environment-markers": {
             "implementation_name": "cpython",
             "implementation_version": "3.6.4",
-            "os_name": "nt",
-            "platform_machine": "AMD64",
+            "os_name": "posix",
+            "platform_machine": "x86_64",
             "platform_python_implementation": "CPython",
-            "platform_release": "10",
-            "platform_system": "Windows",
-            "platform_version": "10.0.15063",
+            "platform_release": "4.14.20-1-lts",
+            "platform_system": "Linux",
+            "platform_version": "#1 SMP Sat Feb 17 21:35:28 CET 2018",
             "python_full_version": "3.6.4",
             "python_version": "3.6",
-            "sys_platform": "win32"
+            "sys_platform": "linux"
```

Comment:
Bleh, I don't like how pipenv does all this, I wish it didn't even include this information. Oh well, it's not the worst, just annoying.